### PR TITLE
Documentation updates

### PR DESCRIPTION
**Pull request by alexandrebodin**
#### Description of what you did:

Documentation updates.

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [ ] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [ ] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [ ] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: docs/3.0.0-beta.x/guides/queries.md [hunk 1]
```diff
@@ -1,406 +1,287 @@
...
-    async create(values) {
-      const relations = _.pick(
-        values,
-        model.associations.map(ast => ast.alias)
-      );
-      const data = _.omit(values, model.associations.map(ast => ast.alias));
+```js
+strapi
+  .query('post')
+  .findOne({ title: 'my title', created_at: '2019-01-01T00:00:00Z' });
```
Comment (soupette) [inline docs/3.0.0-beta.x/guides/queries.md]: If think we need to specify here that the `created_at` field name depends on the db which can also be `createdAt` with Mongo

So here's my proposal

***Using bookshelf***
```js
strapi
  .query('post')
  .findOne({ title: 'my title', created_at: '2019-01-01T00:00:00Z' });
```
***Using mongoose***
```js
strapi
  .query('post')
  .findOne({ title: 'my title', createdAt: '2019-01-01T00:00:00Z' });
```

File: docs/3.0.0-beta.x/guides/queries.md [hunk 2]
```diff
@@ -1,406 +1,287 @@
...
+### `create`
+
+Creates an entry in the database and returns the entry.
+
+#### Example
+
+```js
+strapi.query('post').create({
+  title: 'Mytitle',
+  // this is a group field. the order is persisted in db.
```
Comment (soupette) [inline docs/3.0.0-beta.x/guides/queries.md]: I like this example!

File: docs/3.0.0-beta.x/guides/queries.md [hunk 3]
```diff
@@ -1,406 +1,287 @@
...
+    title: 'Mytitle',
   }
-};
+);
 ```
 
-### Mongoose
+### `delete`
+
+Deletes and entry and return it's value before deletion.
```
Comment (soupette) [inline docs/3.0.0-beta.x/guides/queries.md]: Deletes an entry and returns its value before deletion.

File: docs/3.0.0-beta.x/guides/queries.md [hunk 4]
```diff
@@ -1,406 +1,287 @@
...
-      // Create relational data and return the entry.
-      return model.updateRelations({ _id: entry.id, values: relations });
-    },
+### `search`
 
-    async update(params, values) {
-      // Extract values related to relational data.
-      const relations = _.pick(values, assocs);
-      const data = _.omit(values, assocs);
+Returns entries base on a search on all fields allowing it. (this feature will return all entries on sqlite).
```
Comment (soupette) [inline docs/3.0.0-beta.x/guides/queries.md]: Returns the entries based on a search on all

File: docs/3.0.0-beta.x/guides/queries.md (hunk 1 above)
Comment (derrickmehaffy) [reply docs/3.0.0-beta.x/guides/queries.md]: And also the fact that these fields can be renamed by the user as well.

**Pull request merged**